.github/workflows/migrations.yml

```
@@ -0,0 +1,118 @@
+        image: postgres:16
+        env:
+          POSTGRES_USER: flexprice
+          POSTGRES_PASSWORD: flexprice123
+          POSTGRES_DB: postgres
+        ports: [ '5440:5432' ]
+        options: >-
+          --health-cmd "pg_isready -U flexprice"
+          --health-interval 5s --health-timeout 5s --health-retries 10
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)